README.md

```diff
@@ -12,28 +12,20 @@ Powered by [uniffi-bindgen-react-native] and [create-react-native-library].
 
 ## Installation
 
+### Installation into your own project
+
 This package is available in the [npm registry].
 
 ```sh
 npm i @unomed/react-native-matrix-sdk
 yarn add @unomed/react-native-matrix-sdk
 ```
 
-You also have to install [uniffi-bindgen-react-native]. This will only be necessary until its
-Pod has been released. Make sure that the SHA matches the release of react-native-matrix-sdk.
+You also have to install [uniffi-bindgen-react-native]. 
 
 ```sh
```

Comment:
```suggestion
```

We can actually delete this entirely because on `main` I just updated `uniffi-bindgen-react-native` to its first public release which also does the pod auto linking.